### ISSUE TITLE
Train and Test

### ISSUE THREAD

**Issue by AgustinaLazzati** [laurabolta/Synthesis-Project-I#17]
This component manages the process of training and testing the model. The dataset is first split into training and testing subsets to ensure a fair evaluation. The model is trained on the training set to learn patterns related to student performance, and then evaluated on the test set.
